.github/workflows/cont_integration.yml

```diff
@@ -12,7 +12,7 @@ jobs:
         rust:
           - version: stable
             clippy: true
-          - version: 1.57.0 # MSRV
+          - version: 1.63.0 # MSRV
```

Comment:
Just a heads-up and to link the PRs.
This depends on #1183

Reply:
Yup I'll remove these commits and rebase on #1183 after it's merged.